Formula/docker-machine-driver-xhyve.rb

```diff
@@ -0,0 +1,32 @@
+class DockerMachineDriverXhyve < Formula
+  desc "Xhyve driver for Docker Machine"
+  homepage "https://github.com/zchee/docker-machine-driver-xhyve"
+  url "https://github.com/zchee/docker-machine-driver-xhyve/archive/v0.2.0.tar.gz"
+  sha256 "affba9fe235e0ceb2711a52ea15b380bdcbec8aa2c106b54f750ede3c762dcb8"
+
+  depends_on "go" => :build
```

Comment:
@saljam Thanks reply :)

No, `docker-machine-driver-xhyve` was embeded `xhyve` source.
Not need `xhyve` binary.

See https://github.com/zchee/docker-machine-driver-xhyve/releases/tag/v0.2.0 in Feature.